crates/protogen/proto/rpcsrv/simple.proto

```diff
@@ -0,0 +1,59 @@
+// Definitions for the "simple query interface".
+//
+// This interface provides a way for an external service (Cloud) to submit
+// potentially long running queries to a glaredb database. There is no guarantee
+// about the lifetime of the session used for executing these queries.
+//
+// Design decisions/requirements:
+//
+// - No authentication overhead or needing to go through proxies.
+// - No need to get to response data for a query, just needs to know if the
+//   query succeeded/failed.
+// - Not exposed to the user in any way.
+//
+// Note that this will likely be replaced by a FlightSQL implementation (which
+// would be exposed to the user, unlike what we're doing here).
+//
+// Design docs/rationale:
+// https://github.com/GlareDB/meta/blob/main/notes/2023-11-25.simple-query-interface.md
+
+syntax = "proto3";
+
+import "rpcsrv/common.proto";
+
+package rpcsrv.simple;
+
+message ExecuteQueryRequest {
+  // Info needed to create the session.
+  common.SessionStorageConfig config = 1;
+
+  // Database id we're executing the query for.
+  bytes database_id = 2;
+
+  // The sql query to execute.
+  // TODO: Multiple commands?
+  string query_text = 3;
+}
+
+message QueryResultSuccess {}
+
+message QueryResultError {
+  // The error message for why the query failed.
+  string msg = 1;
+}
+
+// Result of the query execution.
+//
+// TODO: We may want to extend this to actually stream/return the record
+// batches.
```

Comment:
**Question(if-minor)**: Is this something we should hold off until we change to Flight?

Reply:
We don't need to. Depends on the requirements of cloud. It would be straightforward to add the batches to this if cloud needs the results (we already have all the types needed). It would just be a cloud to know how to deserialize those batches (which it'd also need for flightsql anyways).